x-pack/plugins/ml/public/shared_components/single_metric_viewer/single_metric_viewer.tsx

```diff
@@ -204,6 +204,8 @@ const SingleMetricViewerWrapper: FC<SingleMetricViewerPropsWithDeps> = ({
     }
   };
 
+  const timeseriesExplorerStyles = getTimeseriesExplorerStyles();
```

Comment:
Could this be done once outside the component instead of during every render?

Reply:
Done in  [0d5af5f](https://github.com/elastic/kibana/pull/195259/commits/0d5af5f5e20ed4cb007068523929eb0079e7e25a)